nextcord/message.py

```diff
@@ -1269,6 +1269,9 @@ def system_content(self):
         if self.type is MessageType.guild_invite_reminder:
             return "Wondering who to invite?\nStart by inviting anyone who can help you build the server!"
 
+        if self.type is MessageType.role_subscription_purchase:
+            ...  # TODO: implement this
```

Comment:
![image](https://user-images.githubusercontent.com/34910574/210608588-9fb5e2b1-9f06-4717-99a1-5cc08bffb609.png)

Reply:
Thanks! but do you maybe also have the one for a renewal?

Reply:
Taken from datamining: https://paste.nextcord.dev/?id=1686932842152713